terra-jupyter-aou/Dockerfile

```diff
@@ -26,34 +26,35 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
 
 # Spark setup.
 # Copied from terra-jupyter-hail; keep updated.
-
-# Note Spark and Hadoop are mounted from the outside Dataproc VM.
-# Make empty conf dirs for the update-alternatives commands.
-RUN mkdir -p /etc/spark/conf.dist && mkdir -p /etc/hadoop/conf.empty && mkdir -p /etc/hive/conf.dist \
-    && update-alternatives --install /etc/spark/conf spark-conf /etc/spark/conf.dist 100 \
-    && update-alternatives --install /etc/hadoop/conf hadoop-conf /etc/hadoop/conf.empty 100 \
-    && update-alternatives --install /etc/hive/conf hive-conf /etc/hive/conf.dist 100
-
-ENV HAIL_VERSION=0.2.74
 ENV PIP_USER=false
-
-# For dataproc clusters, this path with will be automatically mounted. Else,
-# this is effectively ignored. On GCE VMs, this will result in failures to
-# import the pyspark package.
 ENV PYTHONPATH $PYTHONPATH:/usr/lib/spark/python
 ENV PYSPARK_PYTHON=python3
+ENV HAIL_VERSION=0.2.85
```

Comment:
I think you want to bump it to 0.2.91 now...there has been some bug fixes

Reply:
Done, thanks